src/actor.rs

```diff
@@ -116,22 +117,7 @@ impl Error for CreateError {
 
 impl fmt::Display for CreateError {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        match *self {
-            CreateError::Panicked => f.write_str(&self.to_string()),
-            CreateError::System => f.write_str(&self.to_string()),
-            CreateError::InvalidName(ref name) => {
-                f.write_str(&format!("{} ({})", self.to_string(), name))
-            }
```

Comment:
will this information still be retained?

Reply:
Yes